src/controllers/ArtsController.js

```diff
@@ -150,20 +153,29 @@ class ArtsController {
       }
 
       const {
-        title, description, categoryId, media,
+        title, description, categoryId,
       } = req.body;
 
+
+      let { media } = req.body;
+
+      if (!media) {
+        media = `[{
+        "url":"${process.env.DEFAULT_ARTICLE_IMAGE}",
+        "extension":"jpeg"}]`;
+      }
+
       const mediaFilesArray = JSON.parse(media);
 
-      req.check('title', 'Title is required').notEmpty();
-      req.check('description', 'Description should be longer').notEmpty()
+      req.check('title', 'Title is required').trim().notEmpty();
```

Comment:
Why should I be forced to edit the title and the description?

Reply:
The reason is that whenever an artist wants to update an art, he receives the full payload of details about the art and can therefore decide to make changes to the payload, then send back the payload to the server.
So this will prevent the artist from wiping out either title or description completely.

Reply:
But I can also decide to edit the title and not the description. Or just the description and not the title. There's a way to do that with sequelize